webapp/README.txt

```diff
@@ -0,0 +1,89 @@
+Maintenance instructions, to run when git repository is updated:
```

Comment:
I don't have time to test the webapp. I'll assume it works because it's at bolides.seti.org.